src/OrbitBase/WindowsVersion.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitBase/WindowsVersion.h"
+
+#include <Windows.h>
+#include <Winnt.h>
+#include <absl/strings/str_format.h>
+
+#include "OrbitBase/GetProcAddress.h"
+#include "OrbitBase/StringConversion.h"
+
+namespace orbit_base {
+
+typedef long NTSTATUS;
+#define STATUS_SUCCESS ((NTSTATUS)0x00000000L)
+
+ErrorMessageOr<WindowsVersion> GetWindowsVersion() {
+  static auto rtl_get_version = orbit_base::GetProcAddress<NTSTATUS(WINAPI*)(PRTL_OSVERSIONINFOW)>(
```

Comment:
This is just me not being familiar with Windows: Why don't we call RtlGetVersion directly? Is this a workaround for Windows versions not supporting it?

Reply:
This is because the header is part of the Driver Development Kit (DDK) which we don't necessarily have. This is similar to [llvm's implementation](https://github.com/llvm-mirror/llvm/blob/2c4ca6832fa6b306ee6a7010bfb80a3f2596f824/lib/Support/Windows/Process.inc#L466).

Reply:
I see, thanks!